Publishing SSSOM mapping sets is broken

Trying to get the ODK to publish the SSSOM sets that it manages, by adding the `release_mappings: True` option to the `sssom_mappingset_group` section in the config file, leads to the following error when running `make prepare_release`:

```sh
rsync -R   sssomo.owl   sssomo-base.owl   sssomo-full.owl    ../mappings/modmini-onta.sssom.tsv  ../mappings/modmini-ontb.sssom.tsv  ../mappings/modmini-ontc.sssom.tsv  ../mappings/modmini-ontd.sssom.tsv ../.. &&\
rm -f   sssomo.owl   sssomo-base.owl   sssomo-full.owl  tmp/merged-sssomo-edit.owl tmp/sssomo-preprocess.owl &&\
echo "Release files are now in ../.. - now you should commit, push and make a release \
        on your git hosting site such as GitHub or GitLab"
found ".." dir in relative path: ../mappings/modmini-onta.sssom.tsv
rsync error: syntax or usage error (code 1) at flist.c(2358) [sender=3.2.7]
rsync error: received SIGUSR1 (code 19) at main.c(1613) [Receiver=3.2.7]
make: *** [Makefile:300: prepare_release] Error 1
```

This is because the mappings are in `src/mappings`, while the `rsync` command that does the bulk of copying the release assets to the top-level directory is run from within the `src/ontology` directory. As a result, we are ultimately trying to do `rsync -R [other release files] ../mappings/<mappingset>.sssom.tsv ../..`, which `rsync` does not like at all – the `-R` option is incompatible with some source files being in any directory _above_ the current directory.

Possible solutions:

1) Moving the mappings directory from `src/mappings` to `src/ontology/mappings`. Easiest solution, but a breaking change. And whether it is desirable to have the mappings in `src/ontology` is debatable.

2) Update the `prepare_release` rule to have a separate command that copies the mappings to the top-level directory (similar to what we already do for the patterns). This involves making sure the mappings to be released are _not_ listed in the general `RELEASE_ASSETS` variable, but are instead listed in a separate variable (again, similar to what we do for the patterns).

--- a/template/src/ontology/Makefile.jinja2
+++ b/template/src/ontology/Makefile.jinja2
@@ -340,8 +340,7 @@ ASSETS = \
 RELEASE_ASSETS = \
   $(MAIN_FILES) {% if project.import_group is defined %}{% if project.import_group.release_imports %}$(IMPORT_FILES) {% endif %}{% endif %}\
   $(SUBSET_FILES){% if project.robot_report.release_reports %} \
-  $(REPORT_FILES){% endif %}{% if project.sssom_mappingset_group is defined %}{% if project.sssom_mappingset_group.release_mappings %} \
-  $(MAPPING_FILES){% endif %}{% endif %}
+  $(REPORT_FILES){% endif %}
 
 .PHONY: all_assets
 all_assets: $(ASSETS) {% if project.ensure_valid_rdfxml %}check_rdfxml_assets{% endif %}
@@ -371,7 +370,8 @@ CLEANFILES=$(MAIN_FILES) $(SRCMERGED) $(EDIT_PREPROCESSED)
 
 .PHONY: prepare_release
 prepare_release: all_odk
-	rsync -R $(RELEASE_ASSETS) $(RELEASEDIR) &&\{% if project.use_dosdps %}
+	rsync -R $(RELEASE_ASSETS) $(RELEASEDIR) &&\{% if project.sssom_mappingset_group is defined %}{% if project.sssom_mappingset_group.release_mappings %}
+	mkdir -p $(RELEASEDIR)/mappings && cp -rf $(MAPPING_FILES) $(RELEASEDIR)/mappings &&\{% endif %}{% endif %}{% if project.use_dosdps %}
 	mkdir -p $(RELEASEDIR)/patterns && cp -rf $(PATTERN_RELEASE_FILES) $(RELEASEDIR)/patterns &&\{% endif %}
 	rm -f $(CLEANFILES) &&\
 	echo "Release files are now in $(RELEASEDIR) - now you should commit, push and make a release \
@@ -379,7 +379,8 @@ prepare_release: all_odk
 
 .PHONY: prepare_initial_release
 prepare_initial_release: all_assets
-	rsync -R $(RELEASE_ASSETS) $(RELEASEDIR) &&\{% if project.use_dosdps %}
+	rsync -R $(RELEASE_ASSETS) $(RELEASEDIR) &&\{% if project.sssom_mappingset_group is defined %}{% if project.sssom_mappingset_group.release_mappings %}
+	mkdir -p $(RELEASEDIR)/mappings && cp -rf $(MAPPING_FILES) $(RELEASEDIR)/mappings &&\{% endif %}{% endif %}{% if project.use_dosdps %}
 	mkdir -p $(RELEASEDIR)/patterns && cp -rf $(PATTERN_RELEASE_FILES) $(RELEASEDIR)/patterns &&\{% endif %}
 	rm -f $(patsubst %, ./%, $(CLEANFILES)) &&\
 	cd $(RELEASEDIR) && git add $(RELEASE_ASSETS)
@@ -1244,7 +1245,7 @@ public_release:
 	$(GITHUB_RELEASE_PYTHON) --release $(TAGNAME) $(RELEASEFILES)
 {% else  %}
 
-RELEASE_ASSETS_AFTER_RELEASE=$(foreach n,$(RELEASE_ASSETS), ../../$(n))
+RELEASE_ASSETS_AFTER_RELEASE=$(foreach n,$(RELEASE_ASSETS), ../../$(n)){% if project.sssom_mappingset_group is defined %}{% if project.sssom_mappingset_group.release_mappings %} $(foreach n,$(MAPPING_FILES), ../$(n)){% endif %}{% endif %}
 GHVERSION=v$(VERSION)
 
 .PHONY: public_release

